core/src/main/java/net/adoptopenjdk/icedteaweb/client/commandline/CommandLine.java

```diff
@@ -78,11 +77,11 @@
     public static final int ERROR = 1;
     public static final int SUCCESS = 0;
 
-    public final String PROGRAM_NAME;
+    private final String PROGRAM_NAME;
```

Comment:
use camelCase variable name as it is not `static final`

Reply:
fixed